build/service/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM alpine:3.7
+
+
+# --- Install packages we know we'll need
+RUN apk add --no-cache curl openjdk8
```

Comment:
Part of me feels we should install a specific version of java as well.

I've been burned by java changes in the past.